launchpad/launchpad_reward.gno

```diff
@@ -32,13 +35,17 @@ func init() {
 //
 // returns collected reward amount
 func CollectRewardByProjectId(projectId string) uint64 {
+	common.IsHalted()
+	en.MintAndDistributeGns()
+
 	project, exist := projects[projectId]
 	if !exist {
-		println("NO PROJECT FOR THIS ID", projectId)
+		// println("NO PROJECT FOR THIS ID", projectId)
```

Comment:
Is this used? if not you can delete

Reply:
left over, cleaned up d52e746a8096c94641a42388c350c2b59bf4775a